babel.config.js

```diff
@@ -1,8 +1,21 @@
 module.exports = {
   presets: [
-    '@babel/preset-env',
+    [
+      '@babel/preset-env',
+      {
+        loose: true,
```

Comment:
what is this for?

Reply:
I added this to resolve the Storybook error.
It serves the purpose of modifying certain transpile options in Babel, resulting in some differences in the generated code.

But we did not use the 'loose' option, so I removed it, and when there are plugins included in preset-env but not explicitly added, it can cause these errors. I've found and applied these changes to resolve the issue.
https://github.com/babel/babel/issues/11622#issuecomment-638609015

66b488a